app/models/search/tag_indexer.rb

```diff
@@ -69,7 +69,8 @@ def document(object)
       has_posted_works: object.has_posted_works?,
       tag_type: object.type,
       uses: object.taggings_count_cache,
-      unwrangled: object.unwrangled?
+      unwrangled: object.unwrangled?,
+      filterable: object.synonymous? || object.canonical
```

Comment:
We shouldn't call this field `filterable`, because there exists a definition of `unfilterable` that takes into account other factors:

https://github.com/otwcode/otwarchive/blob/775ed1341c628f91fb345e12b12e5f62e08810b8/app/models/tag.rb#L632-L634

Let's simply call it `canonical_or_synonymous`.